lib/storage/resources.go

```diff
@@ -82,3 +85,25 @@ var SupportedGravityResourcesToRemove = []string{
 	KindAlertTarget,
 	KindTLSKeyPair,
 }
+
+// MetadataSchema is a copy of teleport/lib/services.MetadataSchema but with
+// optional 'name' property because some Gravity resources do not require it
+const MetadataSchema = `{
+  "type": "object",
+  "additionalProperties": false,
+  "default": {},
+  "properties": {
+    "name": {"type": "string"},
+    "namespace": {"type": "string", "default": "default"},
```

Comment:
`namespace` attribute is not serialized in `teleservices.Metadata` so setting the default does not have the desired effect.